TeamZeroServer/src/server/DbConnection.java

```diff
@@ -413,17 +419,108 @@ public boolean addNewGroup(String groupName, String creatingUser) throws SQLExce
 			e.printStackTrace();
 			throw e;
 		}
+		LOGGER.log(Level.FINE, "Getting all groups: {0}", allGroups);
 		return allGroups;
 	}
 	
-	//TODO
-	public ArrayList<Group> getSearchedGroups(String search){
-		return new ArrayList<Group>();
+	
+	public ArrayList<Group> getSearchedGroups(String search) throws SQLException{
+		ArrayList<Group> searchedGroups = new ArrayList<Group>();
+		Connection conn = this.connect();
+		try {
+			
+			//TODO fix this - currently it is not getting the members
```

Comment:
Is it still not getting the members?

Reply:
Ah, sorry about that. It is getting the members now, I just forgot to take out the comment

Reply:
Ok, no problem. I'll remove this comment.